src/assets/stylesheets/components/_form-button.scss

```diff
@@ -0,0 +1,8 @@
+.form-button {
+  width: 100%;
+  height: rem(50px);
```

Comment:
@gutakk  Should this be `56px`?
```suggestion
  height: rem(56px);
```
![Screen Shot 2564-06-18 at 19 17 47](https://user-images.githubusercontent.com/1772999/122559758-f249d300-d069-11eb-9ecd-71c01b7873fa.png)

Reply:
Fixed in ccb5f68ba436a2fdf361a53454c2bac0dbac300b. I got the height from wrong Figma component 😅